pkg/proxy/apis/config/v1alpha1/defaults.go

```diff
@@ -60,6 +60,9 @@ func SetDefaults_KubeProxyConfiguration(obj *kubeproxyconfigv1alpha1.KubeProxyCo
 	if obj.IPVS.SyncPeriod.Duration == 0 {
 		obj.IPVS.SyncPeriod = metav1.Duration{Duration: 30 * time.Second}
 	}
+	if len(obj.IPVS.Scheduler) == 0 {
+		obj.IPVS.Scheduler = "rr"
```

Comment:
a constant can still be used here?

Reply:
tks, the "rr" constant can be obtained from "pkg/proxy/apis/config/types.go", I will update this patch.

Reply:
thanks, i guess my point is to use the constant everywhere in .go files.